rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/config/ScanPackagesOption.java

```diff
@@ -28,7 +33,7 @@ public String getName()
     @Override
     public String getLabel()
     {
-        return "Scan Java Packages";
+        return "Java packages to scan.";
```

Comment:
I don't think the others use a hard-stop at the end ("."), so won't this look out of place?